### DISCUSSION TITLE
Welcome to Sign-Up-Form Discussions!

### DISCUSSION THREAD

**Discussion by parvans** [parvans/Sign-Up-Form#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by parvans** [comment]
this is signup form using node.js express.js and templates engines
and we use mongoDB in DataBase

**Reply by GITTOGEORGETHOMAS** [reply]
hi

**Reply by GITTOGEORGETHOMAS** [reply]
Hello Its Good

**Reply by parvans** [reply]
it's fine

**Comment by GITTOGEORGETHOMAS** [comment]
Can you share source code

**Reply by parvans** [reply]
U have the code 
ur on of the collaborator
